app/decorators/course_decorator.rb

```diff
@@ -148,31 +138,32 @@ def alphabetically_sorted_sites
     object.sites.sort_by(&:location_name)
   end
 
-  # def preview_site_statuses
-  #   site_statuses.select(&:new_or_running?).sort_by { |status| status.site.location_name }
-  # end
+  def preview_site_statuses
+    object.site_statuses.new_or_running.sort_by { |status| status.site.location_name }
+  end
 
   def has_site?(site)
     !course.sites.nil? && object.sites.any? { |s| s.id == site.id }
   end
 
-  # def sites
-  #   alphabetically_sorted_sites.pluck(:id)
-  # end
-
-  # def funding_option
-  #   if salaried?
-  #     "Salary"
-  #   elsif excluded_from_bursary?
-  #     "Student finance if you’re eligible"
-  #   elsif has_scholarship_and_bursary?
-  #     "Scholarships or bursaries, as well as student finance, are available if you’re eligible"
-  #   elsif has_bursary?
-  #     "Bursaries and student finance are available if you’re eligible"
-  #   else
-  #     "Student finance if you’re eligible"
-  #   end
-  # end
+  # rubocop:disable Lint/DuplicateBranch: Duplicate branch body detected
+  def funding_option
+    # TODO: Fix this typical railsy rubbish
```

Comment:
🙈

Reply:
Oh right.
Forgot about this was going to seek second opinion as the linter was going overboard. It reads clear enough given there is ranking in place.